src/packages/shared-utils/package-actions.ts

```diff
@@ -0,0 +1,56 @@
+import { Action, CognitoUserAttributes, OsMainSourceItem, SEATOOL_STATUS } from "../shared-types";
+import { getLatestRai } from './rai-helper'
+import { isCmsWriteUser } from './user-helper'
+import { isStateUser } from "./user-helper";
+
+export const packageActionsForResult = (
+    user: CognitoUserAttributes,
+    result: OsMainSourceItem
```

Comment:
Great idea moving this logic outside of an API handler; good use of our shared-utils.